src/main/kotlin/kafka/Melding.kt

```diff
@@ -45,3 +45,20 @@ data class Sykmeldingsperiode(
     val fom: LocalDate,
     val tom: LocalDate,
 )
+
+@Serializable
+@SerialName("Inntektsmeldings")
+data class Inntektsmelding(
+    val forespoerselId: UUID,
+    val innsendingsId: UUID,
```

Comment:
Jeg tror vi kaller denne for `innsendingId` i LPS-api-appen?
```suggestion
    val innsendingId: UUID,
```